src/selectedtests/app/project_test_mapping_work_item.py

```diff
@@ -1,25 +1,26 @@
 """Model of Evergreen TestMapping that needs to be analyzed."""
 import structlog
 
-from datetime import datetime
+from datetime import datetime, timedelta
 from pymongo.errors import DuplicateKeyError
 from pymongo import IndexModel, ASCENDING
 
 LOGGER = structlog.get_logger()
-WORK_ITEM_TTL = 14 * 24 * 60 * 60  # 2 Week TTL.
+WORK_ITEM_TTL = timedelta(weeks=2).total_seconds()
 
 
 def setup_indexes(collection):
     """
-    Create appropriate indexes for TestMappingWorkItems.
+    Create appropriate indexes for ProjectTestMappingWorkItemms.
 
     :param collection: Collection to add indexes to.
     """
     index = IndexModel([("project", ASCENDING)], unique=True)
     collection.create_indexes([index])
+    LOGGER.info("Adding indexes for collection", collection=collection.name)
 
 
-class TestMappingWorkItem(object):
+class ProjectTestMappingWorkItemm(object):
```

Comment:
pytest thought this was a test file so I added a "Project" prefix to the class...I think this is more accurate anyway